tox.ini

```diff
@@ -75,8 +75,7 @@ commands =
     python run_pylint.py
 deps =
     pep8
-    -ehg+https://bitbucket.org/logilab/pylint@33e334be064c#egg=pylint
-    logilab-common<=0.63.0  # Hack for pylint upstream
+    pylint
     unittest2
```

Comment:
You should definitely add astroid>=1.4.1 here, this will fix some of the build failures, including the spurious undefined-variable. Sorry for chiming in, I'm trying to find what broke for folks with this release.